Update hostmap.md (#26915)
DataDog/documentation: content/en/infrastructure/hostmap.md

- `field:metadata_machine`
- `field:apps`

> The `field:apps` is dervied from the metrics the Datadog Agent is collecting from that host, and can be used to determine
> what intergrations or Datadog products are running on that host. 
> For example, a value of `field:apps:universal` indicates that Universal Service Monitoring is running on that host. 

Reviewer: ```suggestion
The `field:apps` is derived from the metrics collected by the Datadog Agent on a host, and it indicates which integrations or Datadog products are active. For example, a value of `field:apps:universal` indicates that Universal Service Monitoring is running on that host.
```
